test/tasty/neg/src-2/TestDelayedChained.check

```diff
@@ -1,5 +1,5 @@
 TestDelayedChained_fail.scala:4: warning: match may not be exhaustive.
-It would fail on the following inputs: LeafB(), LeafC()
+It would fail on the following inputs: (_ : tastytest.DelayedChained.Internal.Branch), LeafB(), LeafC()
```

Comment:
Is this really desired?

```
sealed trait A
sealed trait B extends A
object U extends A
object V extends B

object Test {
  def foo(x: A) = x match { case U => 0 }
}
```

now gives

```
Test.scala:7: warning: match may not be exhaustive.
It would fail on the following inputs: (_ : B), V
  def foo(x: A) = x match { case U => 0 }
                  ^
1 warning
```

before it would only list `V` as counter-example.

Reply:
Yeah, I think it's not.

What do you think of the original case?  Do you think an uninhabited sealed trait should be unreachable?

Next time I rotate onto this I'll see if it's possible to get this to not warn unreachable but also not make your case warn in-exhaustive for the sealed trait.